changelog.txt

```diff
@@ -1,8 +1,10 @@
 *** Changelog ***
 
 = 8.2.0 - xxxx-xx-xx =
-- Tweak - Remove the functionality for saving the customized statement descriptors.
+* Add - Enable custom styling of the Payment Elements for stores using the updated checkout experience.
+* Tweak - Remove the functionality for saving the customized statement descriptors.
```

Comment:
This line is just fixing up a previous changelog that used the wrong bullet point character.

Reply:
The same as the change in `readme.txt`